Vector/Model/RoomList/RecentsDataSource.m

```diff
@@ -102,6 +102,9 @@ - (MXKSessionRecentsDataSource *)addMatrixSession:(MXSession *)mxSession
         // Replace default event formatter
         recentsDataSource.eventFormatter = [[EventFormatter alloc] initWithMatrixSession:mxSession];
         recentsDataSource.eventFormatter.isForSubtitle = YES;
+
+        // And default room summary updater
```

Comment:
Same remark as MatrixKit PR, we should handle this event formatter at account level...